h2o-core/build.gradle

```diff
@@ -13,9 +13,11 @@ dependencies {
   compile "gov.nist.math:jama:1.0.3"
   compile 'org.javassist:javassist:3.18.2-GA'
   compile "org.apache.commons:commons-math3:3.3"
+  compile "commons-codec:commons-codec:1.9"
```

Comment:
Is this dependency needed?

Reply:
yes this is introduced because class `Credentials` was extracted from `JettyHTTPD` (which was duplicated) so that it can be used in Jetty-independent API (H2oServletContainer).
Originally, it used Base64 implementation from Jetty; this is no longer possible, so we use Apache's impl for the same thing. Let me know if you need more details.